src/plugins/gnosis/index.ts

```diff
@@ -0,0 +1,101 @@
+
+/**
+ * Returns the token `method` from a given ERC-20 contract address
+ * @param web3
+ * @param tokenAddress
+ * @param method
+ */
+const getTokenMethod = async (web3, tokenAddress, method) => {
+    return await call(web3, erc20Abi, [
+      tokenAddress,
+      method
+    ]);
+}
+
+export default class Plugin {
+  public author = 'David';
```

Comment:
This param should be your GitHub username
```suggestion
  public author = 'davidalbela';
```